python/ray/dashboard/client/src/pages/dashboard/tune/TuneTable.tsx

```diff
@@ -313,8 +310,8 @@ class TuneTable extends React.Component<
       return null;
     }
 
-    const firstTrial = Object.keys(tuneInfo["trial_records"])[0];
-    const paramsDict = tuneInfo["trial_records"][firstTrial]["params"];
+    const firstTrial = Object.keys(tuneInfo.trial_records)[0];
+    const paramsDict = tuneInfo.trial_records[firstTrial]["params"];
```

Comment:
["params"] should be .params.  For only the lines you've touched, can you replace all property accesses of literal property names to use dot notation instead of bracket notation.